numtrip/numtrip.py

```diff
@@ -0,0 +1,81 @@
+1
+feld= [2,4,2,4,2],[4,2,8,8,8],[4,8,4,4,4],[2,4,4,8,8],[8,4,8,16,32]
+def spielfeld():
+    zeilennummer= 1
+    """
+    print('          A      B      C      D      E')
+    """
+    print('          1      2      3      4      5')
+
+    for zeile in feld:
+        print('      +------+------+------+------+------+')
+        print('      |      |      |      |      |      |')
+        print(f'   {(zeilennummer)}  ', end='')
+        for zelle in zeile:
+            if zelle >= 10000:
+                print(f'| {(zelle)}', end='')
+            elif zelle >= 1000:
+                print(f'| {(zelle)} ', end='')
+            elif zelle >= 100:
+                print(f'|  {(zelle)} ', end='')
+            elif zelle >= 10:
+                print(f'|  {(zelle)}  ', end='')
+            else:
+                print(f'|   {(zelle)}  ', end='')
+        print('|')
+        zeilennummer=zeilennummer+1
+        print('      |      |      |      |      |      |')
+    print('      +------+------+------+------+------+')
+    
+
+spielfeld()
+"""
+def eingabe():
+    #eingabe= input("Geben sie eine Zahl ein: ")
+    x,y=input("Geben sie zwei zahlen ein")
+    try:
+        eingabe = int(eingabe)
+    except:
+        print('Das ist keine Zahl!')
+    
+
+eingabe()
+"""
+def eingabe():
+    
+    x = input('Welche Spalte soll ausgewählt werden? ')
+    try:
+            x = int(x)
+    except:
+        print('Wissen sie nicht was eine Zahl ist?')
```

Comment:
Hier müssen Sie eine Rückgabe initiieren, damit der Code weiter unten nicht doch ausgeführt wird. Da bei dem Aufruf erwartet wird, dass zwei Zahlen zurückkommen (`x,  y = eingabe()`), muss eine Liste zurückgegeben werden

```suggestion
        print('Wissen sie nicht was eine Zahl ist?')
        return [False, False]
```